js/pulse.js

```diff
@@ -272,16 +287,22 @@
                 protocol = getProtocolFromPulseHost(this._pulseHost);
                 pulse_account_name = getPulseAccount(this._pulseHost);
 
-                //Load the Pulse SDK if not already included
+                // Load the Pulse SDK if not already included
                 if(!OO.Pulse){
+                    adModuleState = AD_MODULE_STATE.Loading;
                     amc.loadAdModule(this.name, protocol + pulse_account_name + pulseSDKUrl, _.bind(function(success) {
-                        adModuleJsReady = success;
-                        if(isWaitingForPrerolls){
+                        adModuleState = success ? AD_MODULE_STATE.Ready : AD_MODULE_STATE.Failed;
+                        if(OO.Pulse)
```

Comment:
We should definitely wrap this check with the proper braces `{}`. @pilievOoyala recently discovered that the minifier in the build process omits this style of code ( a conditional with the next line of code assumed as the branch) in the final minified file. This is typically interpreted correctly but there is possibly a bug in the minifier, so to be safe we should add the proper braces here.